tests/Makefile

```diff
@@ -87,54 +98,92 @@ zstd-nolegacy:
 gzstd:
 	$(MAKE) -C $(PRGDIR) zstd HAVE_ZLIB=1 MOREFLAGS="$(DEBUGFLAGS)"
 
+.PHONY:
+zstd-dll :
+	$(MAKE) -C $(ZSTDDIR) libzstd
+
+.PHONY:
+zstd-staticLib :
+	$(MAKE) -C $(ZSTDDIR) libzstd.a
+
+zstdm_%.o : $(ZSTDDIR)/common/%.c
+	$(CC) -c $(CPPFLAGS) $(CFLAGS) $< -o $@
+
+zstdc_%.o : $(ZSTDDIR)/compress/%.c
+	$(CC) -c $(CPPFLAGS) $(CFLAGS) $< -o $@
+
+zstdd_%.o : $(ZSTDDIR)/decompress/%.c
+	$(CC) -c $(CPPFLAGS) $(CFLAGS) $< -o $@
+
+zstdmt%.o : CPPFLAGS += -DZSTD_MULTITHREAD=1
```

Comment:
Should you use `$(MULTITHREAD_CPP)` here?

Reply:
Yes @terrelln , I guess it would feel more logical